src/elements/Order.php

```diff
@@ -1223,7 +1223,7 @@ public function getIsPaid(): bool
      */
     public function getPaidStatus(): string
     {
-        if ($this->getIsPaid() && $this->getTotal() > 0 && $this->getTotalPaid() > $this->getTotal()) {
+        if ($this->getIsPaid() && $this->getTotal() > 0 && round($this->getTotalPaid(), 2) > round($this->getTotal(), 2)) {
```

Comment:
We can't hard code 2 decimal places here. It would be better to ensure that the amounts compared are rounded already. When rounding use the Currency helper, which can round to the correct number of decimal places for that currency: e.g: `Currency::round($paid - $refunded, $this->currency);`

Reply:
Hold off on making another fix as I investigate this one.

Reply:
For commerce 2, add the rounding inside `getTotalPaid()` and remove it's float return.